components/explore/FilterFloatButton.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="fixed-bottom-middle has-min-width">
+    <NeoButton expanded @click.native="openMobileFilters">
+      {{ $t('general.filters')
+      }}<span v-if="numOfActiveFilters">: {{ numOfActiveFilters }}</span>
+    </NeoButton>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { NeoButton } from '@kodadot1/brick'
+import { usePreferencesStore } from '@/stores/preferences'
+
+const route = useRoute()
+const preferencesStore = usePreferencesStore()
+
+const numOfActiveFilters = computed(() => {
```

Comment:
this is becoming a shared functionality shared by FilterMeunButton, this component and BreadCrumbsFilter
please extract it to a composable